### ISSUE TITLE
The beets-2.0.0.tar.gz tarball should have beets-2.0.0 as the base folder

### ISSUE THREAD

**Issue by mbaldessari** [beetbox/beets#5284]
### Problem

When downloading the beets-2.0.0.tar.gz tarball the main folder should be `beets-2.0.0` and not, as it is currently, `beets-1.6.1`:

```sh
$ curl -L -O https://github.com/beetbox/beets/releases/download/v2.0.0/beets-2.0.0.tar.gz && tar xfz beets-2.0.0.tar.gz && ls
beets-1.6.1  beets-2.0.0.tar.gz
```

**Comment by darkshram** [comment]
Also the version reported in egg-info/dist-info is 1.6.1. Versions of the pypi tarball is Ok. Wrong version is only present in Github tarballs.

**Comment by mbaldessari** [comment]
Seems to be fixed in beets-2.1.0. Closing
